.ci/azure/linux.yml

```diff
@@ -177,6 +177,24 @@ jobs:
     displayName: 'IE CAPITests'
     continueOnError: false
 
+  - script: |
+      wget -P tests/samples_smoke_tests/ https://storage.openvinotoolkit.org/repositories/openvino/ci_dependencies/test/samples_smoke_tests_data.zip     
```

Comment:
File is pretty big. try to use caching https://docs.microsoft.com/en-us/azure/devops/pipelines/release/caching?view=azure-devops